### PR TITLE
Bug#1442843 - fixes - added a restarting Openshift section

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://bugzilla.redhat.com/show_bug.cgi?id=1442843
 - Created Restarting OpenShift section
 - Modularized 'Applying Configuration Section' which appears in all cloud docs.

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@cooktheryan @pschiffe 
 Are there any pre/post tasks when restarting OpenShift services for AWS or GCP (e.g. draining and uncordoning the affected node)? https://bugzilla.redhat.com/show_bug.cgi?id=1442843#c6

**Comment by cooktheryan** [comment]
@gaurav-nelson nothing special is required for restarting the ocp services on AWS.

**Comment by pschiffe** [comment]
The same for GCP.

File: install_config/configuring_aws.adoc [hunk 1]
```diff
@@ -276,10 +227,17 @@ key `kubernetes.io/cluster/<name>,Value=<clusterid>` and have the same value for
 $ oc get pv -o json|jq '.items[].spec.awsElasticBlockStore.volumeID'
 ----
 
-. Restart the master services on the master and the node service on all nodes:
+. Restart the master services on the master and the node service on all nodes,  see xref:../../install_config/master_node_configuration#master-node-config-restart-services[Restarting {product-title} services]:
 +
-[source,bash]
+ifdef::openshift-enterprise[]
```
Comment (jianlinliu) [inline install_config/configuring_aws.adoc]: The above line already link user to 'Master and Node Configuration' section, why here are still these command lines for serverice restart. Should we remove them?

**Comment by gaurav-nelson** [comment]
@bfallonf PTAL

File: install_config/topics/applying_configuration_changes.adoc [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Switching from not using a cloud provider to using a cloud provider produces an
+error message. Adding the cloud provider tries to delete the node because the
+node switches from using the *hostname* as the `*externalID*` (which would have
+been the case when no cloud provider was being used) to using the AWS
+`*instance-id*` (which is what the AWS cloud provider specifies). To resolve
+this issue:
+
+.  Log in to the CLI as a cluster administrator.
+. Check and backup existing node labels:
```
Comment (bfallonf) [inline install_config/topics/applying_configuration_changes.adoc]: s/backup/back up

**Comment by bfallonf** [comment]
@gaurav-nelson One nit, but other than that, LGTM :car:

**Pull request merged**